website/docs/reference/manifest.md

```diff
@@ -401,6 +401,25 @@ inlining-strategy = 18
 > If you need to deploy your contracts on Starknet, it is recommended to use the `release` strategy for their compilation.
 > You can use profile settings overwriting, for more granular control of which builds use the `avoid` and `release` strategy.
 
+### `skip-optimizations`
+
+If enabled, Scarb will skip as much compiler optimization as possible when compiling to Sierra.
+Since inlining is an optimization as well, setting this field to `true` will cause inlining to behave
+as if [`inlining-strategy`](#inlining-strategy) was set to `avoid`, regardless of the value of 
```

Comment:
Let's throw an error unless the `inlining-strategy` is unset or explicitly set to `avoid`.